data/json/items/resources/misc.json

```diff
@@ -33,4 +33,20 @@
     "flags": [ "NONCONDUCTIVE" ],
     "use_action": { "type": "deploy_furn", "furn_type": "f_floor_mattress" }
   }
+  },
+  {
+    "id": "down_mattress",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "down_mattress",
+    "name_plural": "down_mattresses",
+    "description": "This is a single, or twin, sized down filled mattress.",
+    "weight": 20000,
+    "volume": "300000 ml",
+    "price": 1000,
+    "material": [ "cotton" ],
+    "symbol": "0",
+    "color": "white",
+    "use_action": { "type": "deploy_furn", "furn_type": "f_feather_mattress" }
```

Comment:
You need to add a furniture entry for this furniture.

Reply:
Thanks! I've created a furniture entry now that mimics the floor mattress with some changes to the deconstruct.